src/mpid/ch3/src/ch3u_win_fns.c

```diff
@@ -380,6 +372,68 @@ int MPID_Win_set_info(MPIR_Win * win, MPIR_Info * info)
         }
     }
 
+    if (info != NULL) {
```

Comment:
Yep. Looks like so. I was going through patch by patch. As I mentioned the later cleaning patch should be meld into this one.